output/http.md

```diff
@@ -275,30 +288,65 @@ Specifies HTTP authentication:
 </auth>
 ```
 
-#### `method`
-
-| type | default | available values | version |
-| :--- | :--- | :--- | :--- |
-| enum | basic | basic | 1.7.0 |
-
-The method for HTTP authentication. Now only `basic`.
-
-#### `username`
+##### `username`
 
 | type | default | version |
 | :--- | :--- | :--- |
 | string | nil | 1.7.0 |
 
 The username for basic authentication.
 
-#### `password`
+##### `password`
 
 | type | default | version |
 | :--- | :--- | :--- |
 | string | nil | 1.7.0 |
 
 The password for basic authentication.
 
+#### Parameters for `method aws_sigv4`
+
+Parameters for [AWS Signature Version 4](https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html).
+
+```text
+<auth>
+  method aws_sigv4
+  aws_service osis
```

Comment:
Does this example `osis` is appropriate?  I'm not sure `osis` is famous enough.

Reply:
Thanks for your review!

I brought this example from https://github.com/fluent/fluentd/issues/4444.
I can't judge whether this is appropriate.
If you have a better example, please let me know.

Reply:
Thanks.
In AWS context, it may be obvious.
okay, so keep it as is.

Reply:
OK! Thanks!